docs/variants/odroid_h4/odroid-assembly-guide.md

```diff
@@ -0,0 +1,154 @@
+
+To toggle the power supply use the below command:
+
+```bash
+rte_ctrl rel
+```
+
+### Basic power operations
+
+Basic power operations should be performed based on the `rte_ctrl` script
+implemented in `meta-rte` (OS image dedicated to the RTE platform). To perform
+basic power operations use the below-described commands:
+
+1. Turn on the platform:
+
+    ```bash
+    rte_ctrl pon
+    ```
+
+1. Turn off the platform:
+
+    ```bash
+    rte_ctrl poff
+    ```
+
+> Note, that in order for the above commands to work properly, the platform
+should be powered up: both RTE Relay and the power supply must be connected and
+turned on.
+
+### External flashing
+
+The external flashing procedure should be performed based on the scripts
+implemented on the RTE platform. To perform the flashing operation reproduce,
+the below-described steps:
+
+!!! Danger "Important"
+    It is recommended to detect the chip with the DC jack power cable
+    disconnected from the Odroid. If the chip is successfully detected,
+    you can reconnect the DC power jack and proceed with your task. If the
+    chip fails to detect after reconnecting the power, you may need to attach
+    an additional grounding cable to the RTE. This additional grounding is not
+    always necessary, but it can resolve detection issues if they arise.
+
+1. First run the flashrom command on RTE to check if the flash is detected by
+    following the first three steps from [this guide](https://docs.dasharo.com/transparent-validation/rte/v1.1.0/specification/#how-to-set-gpio-states-to-flash-spi)
+    , this chip requires 3.3V.
+
+1. Next check if the pomona is connected properly.
+
+    ```bash
+    flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000
+    ```
+
+1. By using `scp` put the requested Dasharo file to the RTE:
+
+    ```bash
+    scp <path_to_firmware>/<firmware_file> root@<RTE_IP>:/tmp/coreboot.rom
+    ```
+
+    Where:
+
+    - `path_to_firmware` - path to firmware, which should send to RTE,
+    - `firmware_file` - the name of the firmware file, which should be sent
+        to RTE,
+    - `RTE_IP` - IP address of the connected RTE.
+
+1. Login to RTE via `ssh` or `minicom`.
+1. Read the flash chip by executing the following command on RTE:
+
+    ```bash
+    flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -r /tmp/dump.rom
+    ```
+
+1. If the reading was successful, the output from the command above should
+    contain the phrase `Verifying flash... VERIFIED`.
+1. To be completely sure, verify the `sha256sum` of multiple dumps.
+
+    ```bash
+    sha256sum /tmp/dump*.bin
+    ```
+
+1. Write the flash chip by executing the following command on RTE:
+
+    ```bash
+    flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w /tmp/coreboot.rom
+    ```
+
+    > Do not interrupt the flashing procedure in any way (especially by
+    changing connections). It may cause hardware damage!
+
+1. If the reading was successful, the output from the command above should
```

Comment:
```suggestion
1. If the writing was successful, the output from the command above should
```

Reply:
addressed in latest commit
https://github.com/Dasharo/docs/pull/878/commits/ee0640e340048aef8480a7013fb064660e0d7a6b